nevergrad/benchmark/experiments.py

```diff
@@ -594,23 +580,13 @@ def powersystems(seed: Optional[int] = None) -> Iterator[Experiment]:
 
 
 @registry.register
-def powersystemssplit(seed: Optional[int] = None) -> Iterator[Experiment]:
+def powersystemssplit(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     """Unit commitment problem, i.e. management of dams for hydroelectric planning.
     Bigger budget than the powersystems problem."""
-    funcs: List[ExperimentFunction] = []
-    funcs += [PowerSystem(3)]
-    funcs += [PowerSystem(num_dams=3, depth=5, width=5)]
-    funcs += [PowerSystem(num_dams=3, depth=9, width=9)]
-    funcs += [PowerSystem(5)]
-    funcs += [PowerSystem(num_dams=5, depth=5, width=5)]
-    funcs += [PowerSystem(num_dams=5, depth=9, width=9)]
-    funcs += [PowerSystem(9)]
-    funcs += [PowerSystem(num_dams=9, width=5, depth=5)]
-    funcs += [PowerSystem(num_dams=9, width=9, depth=9)]
-    funcs += [PowerSystem(13)]
-    funcs += [PowerSystem(num_dams=13, width=5, depth=5)]
-    funcs += [PowerSystem(num_dams=13, width=9, depth=9)]
-
+    funcs: tp.List[ExperimentFunction] = []
+    for dams in [3, 5, 9, 13]:
+        for depth_width in [3, 5, 9]:
+            funcs += [PowerSystem(dams, depth=depth_width, width=depth_width)]
```

Comment:
for loops are more efficient for this :D

Reply:
shame on me...